client/libs/ga-scroll-depth.js

```diff
@@ -0,0 +1,70 @@
+// Simplified/rewritten from https://github.com/leighmcculloch/gascrolldepth.js
+
+import { onWindowScrollDebounce$ } from '../js/utils/dom-events';
+const startTime = new Date().getTime();
+const getElHeight = el => el.offsetHeight + el.offsetTop;
+
+const getWindowHeight = () => {
+  return window.innerHeight || document.documentElement.clientHeight || document.body.clientHeight;
+};
+
+const getPageYOffset = () => {
+  return window.pageYOffset;
+};
+
+const sendEvent = (distance, timing) => {
+  window.ga('send', {
+    hitType: 'event',
+    eventCategory: 'Scroll Depth',
+    eventAction: 'Percentage',
+    eventLabel: distance,
+    eventValue: 1,
+    eventNonInteraction: true
+  });
+
+  window.ga('send', {
+    hitType: 'timing',
+    timingCategory: 'Scroll Timing',
+    timingVar: `Milliseconds to ${distance}`,
```

Comment:
That said, it's output in GA as time in seconds, so perhaps e.g. `Scroll 25%` is better.